README.md

```diff
@@ -80,5 +80,3 @@ REST calls to the API.
 - If you have any questions about our projects you can email [support@flagsmith.com](mailto:support@flagsmith.com)
 
 ## Acknowledgements
```

Comment:
Sorry to be a pain - I should have removed this in my previous PR, but can we remove this line as well?

Reply:
```suggestion
```

Reply:
This should work ^